producer/proto/render.go

```diff
@@ -115,6 +118,13 @@ func IPRenderer(msg *ProtoProducerMessage, fieldName string, data interface{}) i
 	return NilRenderer(msg, fieldName, data)
 }
 
+func DateTimeRenderer(msg *ProtoProducerMessage, fieldName string, data interface{}) interface{} {
+	if dataC, ok := data.(uint64); ok {
+		return time.UnixMicro(int64(dataC / 1000)).Format(time.RFC3339Nano)
```

Comment:
Needs to support nanoseconds. Not just micro.

Reply:
ISO time format does not support nano

Reply:
I'm ok adding RFC3339Nano which supports it.
Please provide reasons that you need to truncate to Micro.